src/endpoints/groups/byRoom.js

```diff
@@ -0,0 +1,19 @@
+// Express setup
+const express = require("express");
+const app = express();
+
+const Group = require("../../models/group.js");
+
+function getGroupsByRoom(req, res) {
+	var query = Group.find({ 'room_id': req.params.room_id });
+	query.exec(function (err, groups) {
```

Comment:
The query will automatically execute.  
Also use promises instead of callback functions b/c they are a bit neater.  

```js
var promise = Group.findById(req.params.room_id)
    .then((room) => {
        console.log(`room: ${room}`);
    })
    .catch((err) => {
        throw `error finding room with id: ${err}`;
    });

// Then use a helper to send the result of this promise as an HTTP request
helpers.response.sendPromise(promise);
```